app/views/content/steps-to-become-a-teacher/_step_1_check_your_qualifications.html.erb

```diff
@@ -2,6 +2,8 @@
 
 <p>You also need a degree in any subject to teach primary and secondary – if you have one or an equivalent qualification, you can do postgraduate teacher training.</p>
 
-<p>If you do not have a degree, you can do undergraduate teacher training to get a degree alongside qualified teacher status (QTS).</p>
+<p>If you do not have a degree, you can do undergraduate teacher training to get a degree alongside qualified teacher status (QTS). Or you may be eligible to do a <a href=" https://educationhub.blog.gov.uk/2024/02/04/teacher-degree-apprenticeships-how-they-work-and-when-to-apply/"> Teacher Degree Apprenticeship (TDA)</a> and work in a school, earning a salary while getting a batchelor’s degree and QTS.</p>
```

Comment:
```suggestion
<p>If you do not have a degree, you can do undergraduate teacher training to get a degree alongside qualified teacher status (QTS). Or you may be eligible to do a <a href="https://educationhub.blog.gov.uk/2024/02/04/teacher-degree-apprenticeships-how-they-work-and-when-to-apply/">teacher degree apprenticeship (TDA)</a> and work in a school, earning a salary while getting a bachelor’s degree and QTS.</p>
```

Reply:
@MaxineCl Still need to:

* remove caps from teacher degree apprenticeship
* correct typo - batchelor -> bachelor